src/zkbob/manager/AllowListOperatorManager.sol

```diff
@@ -0,0 +1,132 @@
+        allowListEnabled = _allowListEnabled;
+        bool[] memory _allowed = new bool[](_operators.length);
+        for (uint256 i = 0; i < _operators.length; i++) {
+            _allowed[i] = true;
+        }
+        _setOperators(_operators, _allowed, _feeReceivers);
+    }
+
+    /**
+     * @dev Doesn't return any data, as operator URI is not used in this implementation.
+     */
+    function operatorURI() external pure returns (string memory) {
+        return "";
+    }
+
+    /**
+     * @dev Sets the allow list enabled flag.
+     * @param _allowListEnabled flag to enable or disable allow list.
+     */
+    function setAllowListEnabled(bool _allowListEnabled) external onlyOwner {
+        allowListEnabled = _allowListEnabled;
+        emit UpdateAllowListEnabled(_allowListEnabled);
+    }
+
+    /**
+     * @dev Adds or removes an operator from the allow list.
+     * @param _operator address of the operator.
+     * @param _allowed flag to enable or disable operator.
+     * @param _feeReceiver address of the fee receiver.
+     */
+    function setOperator(address _operator, address _feeReceiver, bool _allowed) external onlyOwner {
+        _setOperator(_operator, _allowed, _feeReceiver);
+    }
+
+    /**
+     * @dev Adds or removes operators from the allow list.
+     * @param _operators addresses of the operators.
+     * @param _allowed flags to enable or disable operators.
+     * @param _feeReceivers addresses of the fee receivers.
+     */
+    function setOperators(
+        address[] calldata _operators,
+        bool[] calldata _allowed,
+        address[] calldata _feeReceivers
+    )
+        external
+        onlyOwner
+    {
+        _setOperators(_operators, _allowed, _feeReceivers);
+    }
+
+    /**
+     * @dev Sets the fee receiver for the operator.
+     * @param _feeReceiver address of the fee receiver.
+     */
+    function setFeeReceiver(address _feeReceiver) external {
+        require(isOperator(msg.sender), "OperatorManager: operator not allowed");
+        operatorFeeReceiver[msg.sender] = _feeReceiver;
+        emit UpdateOperator(msg.sender, _feeReceiver, true);
+    }
+
+    function _setOperators(
+        address[] memory _operators,
+        bool[] memory _allowed,
+        address[] memory _feeReceivers
+    )
+        internal
+    {
+        require(_operators.length == _feeReceivers.length, "OperatorManager: arrays length mismatch");
+        require(_operators.length == _allowed.length, "OperatorManager: arrays length mismatch");
+
+        for (uint256 i = 0; i < _operators.length; i++) {
+            _setOperator(_operators[i], _allowed[i], _feeReceivers[i]);
+        }
+    }
+
+    function _setOperator(address _operator, bool _allowed, address _feeReceiver) internal nonZeroAddress(_operator) {
+        operators[_operator] = _allowed;
+        if (_feeReceiver != address(0) && _allowed) {
+            operatorFeeReceiver[_operator] = _feeReceiver;
```

Comment:
When disabling operator, I think we should also reset `operatorFeeReceiver`.

Reply:
In this case, the fee receiver address will not be able to withdraw fees from the pool contract after the operator has been disabled. But I am not sure about the significance of this feature. I suppose it may not be crucial.

Reply:
You are right. Let's keep it then, so that fee receiver can still withdraw. But let's remove the `_feeReceiver != address(0)` condition then, so that fee receiver is always set when operator is enabled/re-enabled.